lib/mailgun_utils.rb

```diff
@@ -0,0 +1,14 @@
+module MailgunUtils
```

Comment:
Nit-picky but the name feels a bit scuzzy. Can we call this `MailgunHeaders` or something instead?